### DISCUSSION TITLE
Can I have help with installation?

### DISCUSSION THREAD

**Discussion by Ilyasff** [gushmazuko/metasploit_in_termux#279] [Q&A]
When I want to install a tool, when I am about to finish, it writes to me:Could not locate Gemfile
![Screenshot_20240830-151929](https://github.com/user-attachments/assets/e838d35a-bb20-442d-af0c-a8a03b87f5f9)

**Comment by hope3245** [comment]
Try reinstalling at 0.119.
And there's still a bug.
You can check the label by installing the app.
Do not upgrade the package before the Ruby 3.5.0 error.
The system will require a new Gem file.
If an error has already occurred, formatting Android is likely to be the solution.
This is also one of the bugs because it can generate additional payloads but not run on the OS.

**Comment by hope3245** [comment]
![Screenshot_20250404_220849_Termux](https://github.com/user-attachments/assets/8307fdd7-cd8b-470f-853b-c8aca1944d82)

If you install it a little while and upgrade the package now, it will not work as the error appears.
In that case, if you delete it completely and reinstall it, it works well today.
It is better not to upgrade the package.
And if you are also installing the command line, it is better to install it in advance.
This is because the installation of the command line is like upgrading the package.
Anyway, there is no problem now.
